ecf/scripts/gfs/wave/init/jgfs_wave_init.ecf

```diff
@@ -4,8 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:10:00
-#PBS -l place=vscatter,select=1:ncpus=128
-#PBS -l place=excl
+#PBS -l place=vscatter,select=1:mpiprocs=11:ompthreads=1:ncpus=11
```

Comment:
Similar to my comment on the gdas_wave_init script, excl or memory?

Reply:
2GB

Reply:
Resolved in a1ca249a